app/controllers/api/v4/imports_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V4::ImportsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def import
+    return head :not_found unless Flipper.enabled?(:imports_api)
+
+    definition = Api::V4::Imports.import_resource_list.merge(definitions: Api::V4::Imports.all_definitions)
```

Comment:
Can we not do this merge in `Api::V4:Imports`? `import_resource_list` + `all_definitions` = imports right?

Reply:
Makes sense. This operation is done in the Validation layer for other schemas, but I hadn't made a validation class given how small this currently is. I'll create a `schema_with_definitions` in `Api::V4::Imports` for now.